packages/app-project/src/screens/ClassifyPage/components/WorkflowMenu/WorkflowMenu.js

```diff
@@ -17,8 +18,19 @@ export default function WorkflowMenu({
 }) {
   const router = useRouter()
   const { owner, project } = router?.query || {}
+  const [ activeSubjectSet, setActiveSubjectSet ] = useState()
   const [ activeWorkflow, setActiveWorkflow ] = useState(workflowFromUrl)
 
+  function onSelectSubjectSet(event, subjectSet) {
+    const useSubjectSelection = activeWorkflow.id === '16106'
```

Comment:
So the EC Subject Picker only works for _production_ workflow 16106? This works for me - I'm fine with temporary hardcoding - but I think this might benefit from a huge comment block saying `/* Wakka wakka! The Subject Picker only work for specific Engaging Crowds projects right now, because it has an external dependency at subject-set-search-api.zooniverse.org, which requires specific subject sets pre-loaded into the database. We'll fix this some time after Junetober 2021! Wakka wakka! */`

My basic sanity check is that this line (along with the similar line in the _other WorkflowMenu.js_ ) should explain, to any Zooniverse developer unfamiliar with EC, 1. **why** this code only works for _production_ workflow 16106, and 2. **when** such temporary hardcoding should be re-examined.

Reply:
Datasette is set up now, so I think the next PR will remove this in favour of pinging Datasette to check if the active subject uses indexed subjects.

Reply:
Datasette sends back 404 if you request a table that doesn't exist. I'll use that to check whether indexing is supported for real projects.
https://subject-set-search-api.zooniverse.org/subjects/927520.json